tools/code-owners-parser/CodeOwnersParser/Azure.Sdk.Tools.CodeOwnersParser.csproj

```diff
@@ -7,7 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="CommandLine.Net" Version="2.3.0" />
-    <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="7.0.0" />
```

Comment:
What do we gain by depending on this logging package? Is it worth having this extra package dependency?

Reply:
@weshaggard if I remove it, [I cannot use the logger, here](https://github.com/Azure/azure-sdk-tools/blob/121207b09156a63f2274b7cd0d0a10ba238688c2/tools/code-owners-parser/CodeOwnersParser/MatchedCodeOwnerEntry.cs#L55-L67):

![image](https://user-images.githubusercontent.com/4429827/211499857-c96847cb-f5d8-4841-ad3d-410f9c755daa.png)

I used the logger instead of doing `Console.Out.WriteLine`, because I decided to follow the convention established by existing code, as seen in [notification-configuration Program.cs](https://github.com/Azure/azure-sdk-tools/blob/121207b09156a63f2274b7cd0d0a10ba238688c2/tools/notification-configuration/notification-creator/Program.cs#L42-L46), and thus have uniform logging approach.

Does that seem reasonable?

Reply:
I don't know anything about this particular package so I'm not sure what benefits we gain but in general the benefits should be enough to warrant taking an extra dependency (even if that is a MS extension package).

Reply:
@weshaggard I removed this dependency in:
- #5134